spec/cc/engine/content_resolver_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "spec_helper"
+require "rubocop"
+require "cc/engine/content_resolver"
+
+module CC::Engine
+  describe ContentResolver do
+    cops = RuboCop::Cop::Cop.all
+    whitelist = File.read("./spec/support/docs_whitelist.txt").lines.map(&:chomp)
+
+    it "has cops" do
+      expect(cops.count).to eq 310
+    end
+
+    describe "#content" do
+      cops.each do |cop|
+        describe "#{cop.name}" do
+          it "has content" do
+            resolver = ContentResolver.new(cop.name)
+            if !whitelist.include?(cop.name)
```

Comment:
We should probably filter the cops with the whitelist before the `each` above. As-is, you're running a spec that says something like `RuboCop::Cop::Lint::AssignmentInCondition has content`, but is a no-op and does not actually assert that spec has content (which is of course intentional).